Building the server requires svn on the path

Otherwise, the build fails with:

```
[ERROR] Failed to execute goal org.terracotta:maven-forge-plugin:1.2.13:manifest (default) on project terracotta: Execution default of goal org.terracotta:maven-forge-plugin:1.2.13:manifest failed: Execute failed: java.io.IOException: Cannot run program "svn": error=2, No such file or directory -> [Help 1]
```

--- a/pom.xml
+++ b/pom.xml
@@ -37,7 +37,7 @@
 
   <properties>
     <build.edition>opensource</build.edition>
-    <maven-forge-plugin.version>1.2.13</maven-forge-plugin.version>
+    <maven-forge-plugin.version>1.2.20</maven-forge-plugin.version>
     <tc-shader.version>1.2</tc-shader.version>
     <skip.deploy>false</skip.deploy>
 

